File: SpellChecker_CustomDictionary/C#/Form1.cs
using Syncfusion.Text;
using Syncfusion.WinForms.Controls;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace SpellCheckerAdv_dictionary
{
    public partial class Form1 : Form
    {
        SpellChecker spellChecker = new SpellChecker();
        private string BasePath = Path.GetDirectoryName(Application.ExecutablePath) + @"\..\..\CustomDictionary\";
        public Form1()
        {
            InitializeComponent();
            spellChecker.CustomDictionaryPath = BasePath + @"\UK Dictionary.dic";

        }

        private void button1_Click(object sender, EventArgs e)
        {
            spellChecker.SpellCheck(new SpellEditorWrapper(this.richTextBox1));
        }
    }
}

File: SpellChecker_CustomDictionary/VB/Form1.vb
Imports System.IO
Imports Syncfusion.Text
Imports Syncfusion.WinForms.Controls


Partial Public Class Form1

        Private spellChecker As SpellChecker = New SpellChecker()
        Private BasePath As String = Path.GetDirectoryName(Application.ExecutablePath) & "\..\..\CustomDictionary\"
        Private Sub Form1_Load(sender As Object, e As EventArgs)
            spellChecker.CustomDictionaryPath = BasePath + "\UK Dictionary.dic"
        End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        spellChecker.SpellCheck(New SpellEditorWrapper(RichTextBox1))
    End Sub
End Class


